app/controllers/api/v1/pages_controller.rb

```diff
@@ -24,7 +24,7 @@ def show
   def update
     page.assign_attributes(page_params)
 
-    if page.save
+    if FormUpdateService.save_page(page)
```

Comment:
Not sure I follow the difference here,

Reply:
@thomasiles @aliuk2012 and I discussed and agreed a different approach, see [f1061ed](https://github.com/alphagov/forms-api/pull/232/commits/f1061edc2712004b5f1532e8982fae9bf901c3bd).